src/ResourceManager/KeyVault/Commands.KeyVault/help/Remove-AzureRmKeyVault.md

```diff
@@ -27,14 +27,14 @@ Remove-AzureRmKeyVault [-VaultName] <String> [-Location] <String> [-InRemovedSta
 
 ### InputObjectByAvailableVault
 ```
-Remove-AzureRmKeyVault [-InputObject] <PSKeyVault> [[-Location] <String>] [-Force] [-AsJob] [-PassThru]
+Remove-AzureRmKeyVault [-InputObject] <PSKeyVault> [-Force] [-AsJob] [-PassThru]
  [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ### InputObjectByDeletedVault
 ```
-Remove-AzureRmKeyVault [-InputObject] <PSKeyVault> [-Location] <String> [-InRemovedState] [-Force] [-AsJob]
```

Comment:
This parameter is required for recovering a deleted vault.

Reply:
So I have just removed this Parameter for the -Input object parameter set because you can actually obtain the location from the PSVault object.  It remains in the interactive parameter set above.